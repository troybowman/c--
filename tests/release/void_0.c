../decl/void_0.c