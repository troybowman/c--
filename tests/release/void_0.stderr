error: void type is only valid for parameter declarations, line 3
error: variable wrong redeclared at line 7 (previous declaration at line 7)
error: void type is only valid for parameter declarations, line 9
error: void type is only valid for parameter declarations, line 11
error: void type is only valid for parameter declarations, line 13
error: void type is only valid for parameter declarations, line 13
error: void type is only valid for parameter declarations, line 13
error: void type is only valid for parameter declarations, line 20
error: variable wrong redeclared at line 21 (previous declaration at line 21)
error: void type is only valid for parameter declarations, line 23
error: non-void funcion main must return a value
c-- returned nonzero exit code